packages/zoe/src/contracts/auction/assertBidSeat.js

```diff
@@ -0,0 +1,19 @@
+import { assert, details } from '@agoric/assert';
+
+export const assertBidSeat = (zcf, sellSeat, bidSeat) => {
+  const {
+    maths: { Ask: bidMath, Asset: assetMath },
+  } = zcf.getTerms();
+  const minBid = sellSeat.getProposal().want.Ask;
+  const bid = bidSeat.getAmountAllocated('Bid', minBid.brand);
+  assert(
+    bidMath.isGTE(bid, minBid),
+    details`bid ${bid} is under the minimum bid ${minBid}`,
+  );
+  const assetAtAuction = sellSeat.getProposal().give.Asset;
+  const assetWanted = bidSeat.getAmountAllocated('Asset', assetAtAuction.brand);
+  assert(
+    assetMath.isGTE(assetAtAuction, assetWanted),
+    details`more assets were wanted ${assetWanted} than were available ${assetAtAuction}`,
```

Comment:
This will show the brand and value, so if the value is a Nat, it is already helpful. And we have an issue to display brands and other presences better. So I think it makes sense to include now, both for the benefits now and in the future.

Reply:
I think when brands are printed, it shows the structure, but nothing relevant about the brand. I can't visually distinguish 3 moola from 3 simloeans.

Reply:
Yes, that's right, but we are working on that. It doesn't make sense to take out logging of amounts, though, because as I mentioned, the values, if Nats, are still helpful. And, we are actively working on better printing of the brands.

Reply:
> we are actively working on better printing of the brands. 
Thanks, that's the clarification I was looking for.